Assets/MixedRealityToolkit.Extensions/Sharing/Localization/SpatialLocalizer.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.Sharing
+{
+    /// <summary>
+    /// Very simple consumer of <see cref="ISpatialCoordinate"/> to demonstrate usage.
+    /// </summary>
+    public class SpatialLocalizer : MonoBehaviour
+    {
+        [SerializeField]
+        private GameObject targetRoot = null;
```

Comment:
nit: Let's go ahead and add tooltips to serialized fields on their creation to avoid large painful documentation pushes in the future. :)